flocker/node/agents/test/test_blockdevice.py

```diff
@@ -49,6 +53,37 @@
     skip = "flocker.node.agents.blockdevice is only supported on Linux"
 
 
+def mkfs(device):
+    """
+    Synchronously initialize a device file with an ext4 filesystem.
+
+    :param FilePath device: The path to the file onto which to put the
+        filesystem.  Anything accepted by ``mkfs`` is acceptable (including a
+        regular file instead of a device file).
+    """
+    options = []
+    if not device.isBlockDevice():
+        options.extend([
+            # Make the filesystem even if the destination is not a block
+            # device.
+            b"-F",
+        ])
+    command = [b"mkfs"] + options + [b"-t", b"ext4", device.path]
+    run_process(command)
```

Comment:
`run_process` returns a `ProcessResult` . These new helper functions might be more useful if we return that here.